spinnaker/spinnaker_system/google_quota_test.py

```diff
@@ -24,6 +24,7 @@
 
```

Comment:
I'm going to submit this change as a separate PR. Baking in default account credentials doesnt make sense here. The account info should have been passed as command line arguments by the caller (e.g. jenkins task) or left as the default, which is the typical use case for individual developers running from the command line.